ePSXe64Ubuntu.sh

```diff
@@ -25,11 +25,23 @@ tput setaf 1; echo "  Add to Favorites/Lock"; tput sgr0
 tput setaf 1; echo "  CLOSE ePSXe GUI to continue with the script."; tput sgr0
 tput setaf 2; echo "Script started."; tput sgr0
 
+sudo apt-get update
+sudo apt-get -y install wget
```

Comment:
@bryceml want to ask, how come including to install wget when script is initiated using wget?
Could it be that latest patch requires latest version of wget?
Cheers, mate!

```
wget -O ePSXe64Ubuntu.sh https://raw.githubusercontent.com/brandleesee/ePSXe64Ubuntu/master/ePSXe64Ubuntu.sh && bash ePSXe64Ubuntu.sh
```

Reply:
I don't always fetch the script via wget, sometimes I git clone it.  I was making both easier on myself and more foolproof for others by making sure it was installed for the rest of the script.

Reply:
I see. In the same spirit, should we add other packages to fetch besides wget?